apps/server/packages/api/rollup.config.js

```
@@ -0,0 +1,29 @@
+const typescript = require("@rollup/plugin-typescript");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)